src/SupportedEntities.php

```diff
@@ -688,4 +690,24 @@ public static function getFormDisplayInfo($entity_type_id = NULL) {
     return isset($info[$entity_type_id]) ? $info[$entity_type_id] : [];
   }
 
+  /**
+   * Gets Drupal entity type from CiviCRM object name.
+   *
+   * @param $objectName
+   *   The CiviCRM object name.
+   *
+   * @return string|null
+   *   The Drupal entity type, or FALSE if not defined.
+   */
+  public static function getDrupalType($objectName) {
+    $converter = new CamelCaseToSnakeCaseNameConverter();
+    $entityType = 'civicrm_' . $converter->normalize($objectName);
```

Comment:
This is assuming the Drupal entity name is always "civicrm_" plus the snake cased Drupal entity name. Since we're specifying the "civicrm entity name" on the info array, we should actually be using that. This method could statically cache a lookup array built from looping over the info array?